src/Sylius/Behat/Resources/config/services/api.xml

```diff
@@ -70,14 +70,22 @@
 
         <service id="sylius.behat.api_platform_client.product_option" class="Sylius\Behat\Client\ApiPlatformClient" parent="sylius.behat.api_platform_client">
             <argument>product-options</argument>
+            <argument>admin</argument>
         </service>
 
         <service id="sylius.behat.api_platform_client.product_review" class="Sylius\Behat\Client\ApiPlatformClient" parent="sylius.behat.api_platform_client">
             <argument>product-reviews</argument>
+            <argument>admin</argument>
         </service>
 
         <service id="sylius.behat.api_platform_client.product" class="Sylius\Behat\Client\ApiPlatformClient" parent="sylius.behat.api_platform_client">
             <argument>products</argument>
+            <argument>admin</argument>
```

Comment:
Shouldn't you also change the service name?

Reply:
True for all of admin services